data/json/recipes/tools/tools_hand.json

```diff
@@ -263,6 +264,30 @@
     "tools": [ [ [ "drift", -1 ] ] ],
     "components": [ [ [ "2x4", 1 ], [ "stick", 2 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "result": "hammer_crude",
```

Comment:
As far as I know it is not being represented as being cast, it requires an anvil, just not anvil 3. You need a hammer, just not hammer 3. The normal hammer is a modern claw-style hammer, which would require a lot of shaping and notching, which would require an anvil 3. The crude hammer is more of a traditional blacksmith hammer, being a large hardened broad hammer, no claw, no prying, etc. Making such a hammer is far easier than making one that curves.

https://encrypted-tbn1.gstatic.com/shopping?q=tbn:ANd9GcSfeOwZASNr_DSSfKAfluHdaFjYD0eFNAh_92OY89ur6jlPTdeg9yLhJFLu2pA1M53Qyp-8h2iKfRUYT5sfq2FGDqiO6qrQDHb9i_4yVUQS-Fa-H7lgjgtLauBbdRfOKhqsGm1-kqYQAM8&usqp=CAc

https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRUFO6sLGI_fjUOAty2CajYeENx4lbqyUfxvA&usqp=CAU

So basically, it's a simpler hammer, but probably more suited for blacksmithing than the claw hammer would be. You basically take the makeshift hammer, which is a lump of steel on a stick, and you heat it up in a forge and hammer it a lot to harden it. The key is the forge and a flat surface. Why did you think it was cast, just because of the crucible? Because the crucible is used for nearly all smithing currently, despite not being realistically necessary for a lot of forging work.